dbms/src/Flash/Mpp/MPPHandler.cpp

```diff
@@ -27,6 +27,31 @@ extern const char exception_before_mpp_non_root_task_run[];
 extern const char exception_before_mpp_root_task_run[];
 } // namespace FailPoints
 
+namespace
+{
+static void addRetryRegion(const ContextPtr & context, mpp::DispatchTaskResponse * response)
+{
+    for (const auto & table_region_info : context->getDAGContext()->tables_regions_info.getTableRegionsInfoMap())
```

Comment:
`if (!context->isDisaggregatedComputeMode())` Forgotten?

Reply:
> `if (!context->isDisaggregatedComputeMode())` Forgotten?

I will add it.